applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp

```diff
@@ -275,18 +278,27 @@ class MechanicalExplicitStrategy
         TSystemVectorType rDx = TSystemVectorType();
         TSystemVectorType rb = TSystemVectorType();
 
-        // initial operations ... things that are constant over the Solution Step
+        // Initial operations ... things that are constant over the Solution Step
         pScheme->InitializeSolutionStep(r_model_part, matrix_a_dummy, rDx, rb);
 
-        if (BaseType::mRebuildLevel > 0) {
+        if (BaseType::mRebuildLevel > 0) { // TODO: Right now is computed in the Initialize() because is always zero, the option to set the RebuildLevel should be added in the constructor or in some place
```

Comment:
is it much effort to refactor?

Reply:
Well, no, but better not create  a PR with thousand of changes